src/create/CreateContainer.test.tsx

```diff
@@ -5,42 +5,53 @@ import {
   cleanup,
   RenderResult,
 } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import { match } from 'react-router-dom';
 import { CreateContainer } from './CreateContainer';
-import firebase from '../firebase/firebase';
-
-jest.mock('../firebase/firebase', () => ({
-  getUserId: jest.fn(),
-  createBoard: (link: string, rows: number, cols: number) => Promise.resolve(), // eslint-disable-line @typescript-eslint/no-unused-vars
-}));
 
 describe('CreateContainer', () => {
-  beforeEach(() => {
-    jest.spyOn(window.location, 'assign').mockImplementation();
-  });
+  const testHistory = createMemoryHistory();
+  const testMatch: match<string> = {
+    params: '',
+    isExact: false,
+    path: '',
+    url: '',
+  };
 
   afterEach(() => {
-    jest.restoreAllMocks();
-
     cleanup();
   });
 
   it('renders without crashing', () => {
     // given / when / then
-    render(<CreateContainer />);
+    render(
+      <CreateContainer
+        history={testHistory}
```

Comment:
You could pass `testHistory` into an instance of `Router` to avoid passing `react-router` specific props into the container directly, docs - https://testing-library.com/docs/example-react-router

Reply:
Gonna use hooks and wrap with Router :+1: